test/e2e/apps/cronjob.go

```diff
@@ -433,7 +443,7 @@ var _ = SIGDescribe("CronJob", func() {
 		// CronJob resource delete operations
 		expectFinalizer := func(cj *batchv1.CronJob, msg string) {
 			framework.ExpectNotEqual(cj.DeletionTimestamp, nil, fmt.Sprintf("expected deletionTimestamp, got nil on step: %q, cronjob: %+v", msg, cj))
-			framework.ExpectEqual(len(cj.Finalizers) > 0, true, fmt.Sprintf("expected finalizers on cronjob, got none on step: %q, cronjob: %+v", msg, cj))
+			gomega.Expect(cj.Finalizers).To(gomega.BeNumerically(">", 0), fmt.Sprintf("expected finalizers on cronjob, got none on step: %q, cronjob: %+v", msg, cj))
```

Comment:
Beware, a `len` got lost here. As it stands now, a slice gets compared against a number, which wouldn't work.

Normally I'd agree that gomega.BeNumerically is the right choice for such a comparison, but here we are testing for a non-empty slice, so the only case where this fails is `len(cj.Finalizers) == 0`, which is already covered sufficiently by the failure message.

Also, Expect takes a format string, so `fmt.Sprintf` is unnecessary.

=>

```suggestion
			if len(cj.Finalizers) == 0 {
				framework.Failf("expected finalizers on cronjob, got none on step: %q, cronjob: %+v", msg, cj))
			}
```